src/Pkg.jl

```diff
@@ -264,6 +261,47 @@ const develop = API.develop
 #TODO: Will probably be deprecated for something in PkgDev
 const generate = API.generate
 
+"""
+    Pkg.info(; keywords...)::Dict{UUID, PackageInfo}
+    Pkg.info(uuid::UUID; keywords...)::Dict{UUID, PackageInfo}
+    Pkg.info(uuids::Vector{UUID}; keywords...)::Dict{UUID, PackageInfo}
+
+Query the dependecy graph.
+You can choose which packages to query by specyfing a list of uuids or by using *selector* keywords.
+
+`Pkg.info` accepts two classes of keywords **selectors** and **properties**.
+**Selectors** will add certain packages to the set of packages to be queried.
+**Properties** determine which package properties will be queried and written to the `PackageInfo` structs.
+`nothing` is used as the default value of `PackageInfo` fields when no query has been executed.
+
+Selectors:
+
+| `keyword` | `API`                                                                               |
+|:----------|:------------------------------------------------------------------------------------|
+| `active`  | If the active environment is a project, add it to the set of packages to be queried |
+| `direct`  | Add all direct dependecies to the set of packages to be queried                     |
+| `all`     | Add all dependencies to the set of packages to be queried                           |
+
+Properites:
+
+| `keyword`   | `Type`          | `Description`                                      |
+|:------------|:----------------|:---------------------------------------------------|
+| `name`      | `String`        | The name of the package                            |
+| `version`   | `VersionNumber` | The version of the package                         |
+| `developed` | `Bool`          | Whether a package is directly tracking a directory |
+| `pinned`    | `Bool`          | Whether a package is pinned                        |
+| `source`    | `String`        | The directory containing the package's source code |
+
```

Comment:
Which of these will be `nothing` for stdlibs?

Reply:
All fields should "Just Work" for stdlibs, with the exception of `version` which will always be `nothing`.